### ISSUE TITLE
Feature request: get time the request took

### ISSUE THREAD

**Issue by D00mch** [rest-nvim/rest.nvim#147]
With curl I could get it with something like:
`curl -s -w 'Total: %{time_total}s\n' https://google.com`
>>
`... Total: 0.283850s`

How do I get it with rest.nvim?

May be custom curl commands, or flags in setup?

**Comment by aldevv** [comment]
I think this should be a default for every request and response but you can get the time like this, kind of messes with formatting though
```
GET http://localhost:3001/api/product
content-type: application/json

-w '\nTotal: %{time_total}s'
```

**Comment by Liverm0r** [comment]
`I have error running formatter "jq" on response body` with the flag.

**Comment by nvdsalehi** [comment]
> I have `error running formatter "jq" on response body` with the flag.

I managed to append total time as json to the end of the response.
```http
POST {{host}}/test
Content-Type: application/json

-w {"Total":"%{time_total}s"}\n
```
Response would be something like this
```json
{
  "id": 1
}
{
  "total": "0.240195s"
}

```
But body can't be decoded with `local body = context.json_decode(context.result.body)` anymore.

**Comment by Ali-Aref** [comment]
Is there any way to add the time in the header response as follow ???
```http
HTTP/1.1 200 OK
vary: RSC, Next-Router-State-Tree, Next-Router-Prefetch, Next-Url
content-type: application/json
Date: Mon, 15 Jan 2024 05:04:26 GMT
Connection: keep-alive
Keep-Alive: timeout=5
Transfer-Encoding: chunked
Time: 0.240195s <--- here  
```

**Comment by NTBBloodbath** [comment]
I completely forgot to close this issue when I merged #261. Thousand apologies!

You can add the request statistics (time taken, etc) by enabling `show_statistics` in the `result` table in your `rest.nvim` setup. Take a look at the README's example if you need some guidance.

**Comment by pdarulewski** [comment]
It took me a second to figure out how to add that to the configuration. If anyone wants a quick snippet, here it is:

```lua
show_statistics = {
    {
        "time_total",
        title = "Total Time: ",
        type = "time",
    },
}
```
